_studio/shared/src/libmfx_core_vaapi.cpp

```diff
@@ -259,6 +259,8 @@ typedef struct {
     { 0x3EA6, MFX_HW_CFL, MFX_GT3 },
     { 0x3EA7, MFX_HW_CFL, MFX_GT3 },
     { 0x3EA8, MFX_HW_CFL, MFX_GT3 },
+    { 0x3E9C, MFX_HW_CFL, MFX_GT1 },
```

Comment:
This device is unknown to media-driver:
```
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3E9C
[dvrogozh@dvrscl media-driver]$ fgrep -rsn 0x3e9c
media_driver/agnostic/gen10/vp/kernel/igvpkrn_g10.c:41811:    0x70f4964d, 0x3e9c7119, 0xface6797, 0xb08190f5, 0x003ada14, 0x5285fa16, 0x77252f36, 0x76183e56,
media_driver/agnostic/gen10/vp/kernel/igvpkrn_g10.c:44980:    0xcdf9adef, 0xdd3acd73, 0xa644513e, 0x42253a7a, 0x8b2b3872, 0xdaf9e5ec, 0x92c195a2, 0x3e9cf4d4,
media_driver/agnostic/gen11_icllp/vp/kernel/igvpkrn_g11_icllp.c:31341:    0x9cbed3ab, 0x4902ce80, 0x71322923, 0x63cb5b6a, 0x3e9ce47b, 0x33321c19, 0x2d8bca26, 0x75d89b23,
media_driver/agnostic/gen11_icllp/vp/kernel/igvpkrn_g11_icllp.c:36952:    0xdf98d38b, 0x4d1f31d2, 0x3e9c98f5, 0x255e0076, 0xc52e698e, 0x51cc170e, 0xe67ae30a, 0x566f771c,
media_driver/agnostic/gen9/vp/kernel/igvpkrn_g9.c:48393:    0x3e9caf57, 0x7561677c, 0x9eb4b91e, 0xa7ec7677, 0x1cfb0263, 0xd5ac1da8, 0xd70dc1f7, 0x308c2acc,
[dvrogozh@dvrscl media-driver]$
```

Reply:
And it is unknown to Linux kernel: https://github.com/freedesktop/drm-tip/blob/drm-tip/include/drm/i915_pciids.h.

Thus, we should create a separate patch for this ID and avoid merging it in till components we depend from (kernel, media-driver) will not merge in support for this device ID.